documentation/src/test/java/example/ParameterizedTestDemo.java

```diff
@@ -135,9 +136,9 @@ void testWithMultiArgMethodSource(String first, int second) {
 
 	// tag::CsvSource_example[]
 	@ParameterizedTest
-	@CsvSource({ "foo, 1", "bar, 2", "'baz, qux', 3" })
+	@CsvSource({ "foo, 1", "bar, 2", "'baz, qux', 3", "'', 4", ",5" })
 	void testWithCsvSource(String first, int second) {
-		assertNotNull(first);
+		assertNotNull(Objects.toString(first, "<null>"));
```

Comment:
Unless I'm missing something, `Objects.toString(first, "<null>")` can never return `null`.

So, were you perhaps intending to assert something else?